gammapy/detect/_test_statistics.pyx

```diff
@@ -0,0 +1,83 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import print_function, division
+
+import numpy as np
+cimport numpy as np
+cimport cython
+
+cdef np.float_t FLUX_FACTOR
+
+FLUX_FACTOR = 1E-12
+
+
+@cython.cdivision(True)
+@cython.boundscheck(False)
+def _f_cash_root(np.float_t x, np.ndarray[np.float_t, ndim=2] counts,
+                 np.ndarray[np.float_t, ndim=2] background,
+                 np.ndarray[np.float_t, ndim=2] model):
+    cdef np.float_t sum
+    cdef unsigned int i, j, ni, nj
+    ni = counts.shape[1]
+    nj = counts.shape[0]
+    sum = 0
+    for j in range(nj):
+        for i in range(ni):
+            sum += (model[j, i] * (counts[j, i] / (x * FLUX_FACTOR * model[j, i]
+                                                   + background[j, i]) - 1))
+    return sum
+
+
+@cython.cdivision(True)
+@cython.boundscheck(False)
+def __amplitude_bounds(np.ndarray[np.float_t, ndim=2] counts,
```

Comment:
Why double underscore?
I know single underscore means "private" ... if this is roughly the same can you use single underscore here, too?